frontend/src/features/layout/Sidebar/navigation/NavigationMenuList.jsx

```
@@ -0,0 +1,226 @@
+						active
+							? 'text-cinemata-sunset-horizon-400p'
+							: 'text-cinemata dark:text-cinemata-strait-blue-200'
+					)}
+				/>
+			</span>
+		);
+	}
+
+	switch (itemType) {
+		case 'link':
+			const resolvedLinkAttr = { ...(linkAttr || {}) };
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)